Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM hashicorp/terraform:0.12.29
```

Comment:
Looks like this image inherits from alpine linux, just be aware that alpine uses a different libc (musl) than any other linux operating system (except maybe void? and embedded linux solutions). There are some significant differences in the way the networking system calls work in musl vs glibc (the defacto standard). It shoudn't matter too much here, but I'm suspicious of alpine and musl